Testscripts/Windows/DM_HotAddRemove.ps1

```diff
@@ -146,30 +146,26 @@ function Main {
             $timeout -= 1
             Start-Sleep -s 1
         }
-        # stop vm2
-        Stop-VM -VMName $VM2Name -ComputerName $HvServer -force
-        # Verify if errors occured on guest
+        # Verify if errors occurred on guest
```

Comment:
I think we need Stop-VM. Please check again. We had the old log

Reply:
I didn't see the difference for the test result. But if shutdown VM2 there are some time wasting in upload files before fix.
```
11/04/2019 06:04:09 : [DEBUG] Uploading C:\LISAv2\EC82\TestResults\2019-03-11-21-35-39-3337\DYNAMIC-MEMORY-HOT-REMOVE\runtest.sh to USER@IP : 22 using password authentication
11/04/2019 06:04:52 : [WARN ] Error in upload, attempt 1, retrying
11/04/2019 06:05:02 : [DEBUG] Uploading C:\LISAv2\EC82\TestResults\2019-03-11-21-35-39-3337\DYNAMIC-MEMORY-HOT-REMOVE\runtest.sh to USER@IP : 22 using password authentication
11/04/2019 06:05:45 : [WARN ] Error in upload, attempt 2, retrying
11/04/2019 06:05:55 : [DEBUG] Uploading C:\LISAv2\EC82\TestResults\2019-03-11-21-35-39-3337\DYNAMIC-MEMORY-HOT-REMOVE\runtest.sh to USER@ IP : 22 using password authentication
11/04/2019 06:06:39 : [WARN ] Error in upload, attempt 3, retrying
11/04/2019 06:06:49 : [DEBUG] Uploading C:\LISAv2\EC82\TestResults\2019-03-11-21-35-39-3337\DYNAMIC-MEMORY-HOT-REMOVE\runtest.sh to USER@IP : 22 using password authentication
11/04/2019 06:07:32 : [WARN ] Error in upload, attempt 4, retrying
11/04/2019 06:07:42 : [DEBUG] Uploading C:\LISAv2\EC82\TestResults\2019-03-11-21-35-39-3337\DYNAMIC-MEMORY-HOT-REMOVE\runtest.sh to USER@IP : 22 using password authentication
11/04/2019 06:08:26 : [WARN ] Error in upload, attempt 5, retrying
11/04/2019 06:08:36 : [DEBUG] Uploading C:\LISAv2\EC82\TestResults\2019-03-11-21-35-39-3337\DYNAMIC-MEMORY-HOT-REMOVE\runtest.sh to USER@IP : 22 using password authentication
11/04/2019 06:09:19 : [WARN ] Error in upload, attempt 6, retrying
11/04/2019 06:09:29 : [DEBUG] Uploading C:\LISAv2\EC82\TestResults\2019-03-11-21-35-39-3337\DYNAMIC-MEMORY-HOT-REMOVE\runtest.sh to USER@IP : 22 using password authentication
11/04/2019 06:10:12 : [WARN ] Error in upload, attempt 7, retrying
11/04/2019 06:10:23 : [DEBUG] Uploading C:\LISAv2\EC82\TestResults\2019-03-11-21-35-39-3337\DYNAMIC-MEMORY-HOT-REMOVE\runtest.sh to USER@IP : 22 using password authentication
11/04/2019 06:11:06 : [WARN ] Error in upload, attempt 8, retrying
11/04/2019 06:11:16 : [DEBUG] Uploading C:\LISAv2\EC82\TestResults\2019-03-11-21-35-39-3337\DYNAMIC-MEMORY-HOT-REMOVE\runtest.sh to USER@IP : 22 using password authentication
11/04/2019 06:11:59 : [WARN ] Error in upload, attempt 9, retrying
11/04/2019 06:12:09 : [DEBUG] Uploading C:\LISAv2\EC82\TestResults\2019-03-11-21-35-39-3337\DYNAMIC-MEMORY-HOT-REMOVE\runtest.sh to USER@IP : 22 using password authentication
```

Reply:
Fine for now. But I recall we had some conversation about VM destroy without shutdown may cause the network port hang. If we hear any from Azure or Azure networking, then we can add the old command back.